Sources/Fuzzilli/Protobuf/program.proto

```diff
@@ -24,15 +24,13 @@ message Instruction {
     repeated uint32 inouts = 1;
```

Comment:
Ah sorry, but can you undo the reordering here... The idea here is to have them ordered the way they are in Operations.swift and then renumber them (starting from 3 and not 5, then with increasing numbers) once we feel like we have the basic set of operations supported. This would be something to do just before one of the next releases as it will break interoperability.

Reply:
I've reverted the ordering